routing/car_router.cpp

```diff
@@ -225,6 +229,7 @@ IRouter::ResultCode FindSingleOsrmRoute(FeatureGraphNode const & source,
   route.SetTurnInstructions(move(turns));
   route.SetSectionTimes(move(times));
   route.SetStreetNames(move(streets));
+  route.SetTraffic({} /* No traffic info in case of OSRM */);
```

Comment:
Is it ok that in route the invariant you described above is violated? Why not to create a traffic list of Unknowns of size geometry.size() / 2?